nxdialog/Makefile.am

```diff
@@ -0,0 +1,19 @@
+SUBDIRS = bin man
+
+ACLOCAL_AMFLAGS = -I m4
+
+MAINTAINERCLEANFILES =			\
+    $(srcdir)/autom4te.cache/*		\
+    $(srcdir)/build-aux/*		\
+    $(srcdir)/Makefile.in		\
+    $(srcdir)/man/Makefile.in		\
+    $(srcdir)/src/Makefile.in		\
+    $(srcdir)/aclocal.m4		\
+    $(srcdir)/config.h.in		\
+    $(srcdir)/config.h.in~		\
+    $(srcdir)/config.guess		\
+    $(srcdir)/config.sub		\
+    $(srcdir)/configure			\
```

Comment:
there's a tab/space bug in the config.sub line

Reply:
that tab/space flaw is only in the diff, not in the file.